### ISSUE TITLE
@unittest.skip handled incorrectly - test skkipped, but present in output as passed

### ISSUE THREAD

**Issue by Hubbitus** [browncoat-ninjas/nimoy#13]
**Version**: 0.0.1b7

**Reproduction Test Case:**

```
import unittest

from nimoy.specification import Specification


class MinimalSpec(Specification):
    def test_feature_method_pass(self):
        with given:
            a = 'The quick brown fox'
        with expect:
            a == 'The quick brown fox'

    @unittest.skip
    def test_feature_method_fail(self):
        with given:
            a = 'The quick brown fox'
        with expect:
            a == 'The quick Brown Fox'
```

**Expected Behavior:**
`test_feature_method_fail` will not run at all
**Actual Behavior:**
Execution of method skipped, but it present in log as passed:

```
$ nimoy 
test_feature_method_pass (builtins.MinimalSpec) ... ok
test_feature_method_fail (builtins.MinimalSpec) ... ok

----------------------------------------------------------------------
Ran 2 tests in 0.001s

OK
```

**Comment by KVGarg** [comment]
Hi @Hubbitus @noamt is there any reason we're skipping this test?

**Comment by noamt** [comment]
I'm guessing that this test case isn't included in the repo. We can add it

**Comment by KVGarg** [comment]
Can you help me by pointing to the right file, where this test has to be added?

**Comment by noamt** [comment]
@KVGarg I just pushed branch `feature/skipped-spec`. It includes a new spec that reproduces this issue - `specs/nimoy/integration/unittest_integration_spec.py`

**Comment by noamt** [comment]
Fixed with PR #16

**Comment by noamt** [comment]
Will be included in the next release
